### PR TITLE
Update to .NET 7

### PR DESCRIPTION
**Pull request by cmaneu**

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # See here for image contents: https://github.com/devcontainers/images/blob/main/src/dotnet/.devcontainer/Dockerfile
 
 # [Choice] .NET version: 6.0, 3.1, 6.0-bullseye, 3.1-bullseye, 6.0-focal, 3.1-focal
```
Comment (bamurtaugh) [inline .devcontainer/Dockerfile:3]: Should update comment here.

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
 		"args": { 
 			// Update 'VARIANT' to pick a .NET Core version: 3.1, 6.0
```
Comment (bamurtaugh) [inline .devcontainer/devcontainer.json:9]: Should also update this comment.

**Review by bamurtaugh — COMMENTED** [review]
Thanks @cmaneu! Left a couple nit comments 😄

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (cmaneu) [reply .devcontainer/Dockerfile]: I should have seen that the first time 😅

**Comment by cmaneu** [comment]
@bamurtaugh: comments updated :)

**Review by bamurtaugh — APPROVED** [review]
Thanks @cmaneu!

**Pull request merged**